spring-cloud-stream-binder-kinesis-core/src/main/java/org/springframework/cloud/stream/binder/kinesis/properties/KinesisBinderConfigurationProperties.java

```diff
@@ -26,4 +26,13 @@
 @ConfigurationProperties(prefix = "spring.cloud.stream.kinesis.binder")
 public class KinesisBinderConfigurationProperties {
 
+	private boolean autoCreateStreams = true;
+
+	public boolean getAutoCreateStreams() {
+		return autoCreateStreams;
```

Comment:
Hi @artembilan , Thanks for taking the time to review! This implementation is heavily influenced by the Kafka binder - there are a handful of things I questioned myself, but when in doubt I tried to stay consistent with that project.

`autoCreateStreams` is modeled after the Kafka binder's `autoCreateTopics` property [detailed here](http://docs.spring.io/autorepo/docs/spring-cloud-stream-binder-kafka-docs/1.1.0.M1/reference/htmlsingle/#_kafka_binder_properties). I also don't see much value in allowing the binder to fail when there's no existing stream and `autoCreateStreams`=`false`, but it is consistent with the [Kafka binder's handling](https://github.com/spring-cloud/spring-cloud-stream-binder-kafka/blob/master/spring-cloud-stream-binder-kafka-core/src/main/java/org/springframework/cloud/stream/binder/kafka/provisioning/KafkaTopicProvisioner.java#L118). 

Let me know if it's preferable to refactor that property out and default the provisioner to create/update on startup - I'd be all for it.